src/cmdstan/command.hpp

```diff
@@ -1284,15 +990,17 @@ int command(int argc, const char *argv[]) {
           logger, init_writers[0], sample_writers[0], diagnostic_writers[0]);
     }
   }
+  //////////////////////////////////////////////////
```

Comment:
These comments - I think it useful to have something delineating the sections, but I'm not sure it is helpful if they're all the same. Maybe something like 

```cxx
// ------- VARIATIONAL START ------- //
code here;
// ------- VARIATIONAL END ------- //
```

Reply:
good idea.  will do.